perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -544,6 +542,7 @@ sub update_extra_fields : Private {
                         $disable ? (disable => 1, disable_message => $disable_message) : (),
                     }) if $name;
                 }
+                @{$meta->{values}} = sort { $a->{key} <=> $b->{key} } @{$meta->{values}};
```

Comment:
I don't think we can do this, because the order in general is as it comes from the Open311 server, and doing this would change that order. For example, we have many Yes/No answers which have Yes followed by No (as both key and name) , and this would reorder them to No/Yes. We cannot assume that it should be shown in key order.
I think as long as the order shown on the admin web page is maintained, that would be okay, and someone could reorder/rename if needed - the issue is that the sort on line 533 was sorting alphabetically, so the results from the web page would then go 1, 10, 11, ... 2, 3, 4, .... rather than what was entered. If that sort was reinstated but made numerical, then whatever order the user had on the admin page would be kept (and any order created by Open311 import would be maintained).

Reply:
Also the key might not be numerical, it can be anything the council wants. And a regression test to show the admin no longer reorders something with many values would be great.